server/api/project/project.controller.js

```diff
@@ -54,6 +54,22 @@ exports.authors = function(req, res) {
 exports.create = function(req, res) {
   Project.create(req.body, function(err, project) {
     if(err) { return handleError(res, err); }
+
+    //Add creator to any new project
+    var userId = req.params.id;
```

Comment:
I think you want req.user._id. Params doesn't have the user id in it at this point. Did you test it?